quartz/components/scripts/checkbox.inline.ts

```diff
@@ -0,0 +1,29 @@
+import { getFullSlug } from "../../util/path"
+
+const checkboxId = (index: number) => `${getFullSlug(window)}-checkbox-${index}`
```

Comment:
hm sus that this isn't stable but I can't think of anything obviously better (we could hash the content of the page, etc. but those are also brittle and expensive)